src/test/java/com/redhat/fuse/apicurio/jaxrs/tests/GenerateFuseProjectResourceTest.java

```diff
@@ -48,30 +50,43 @@
 	 */
 	private String getRestDSLXMLFromSwagger(String filePath) throws Exception {
 		GenerateFuseProjectResource generator = new GenerateFuseProjectResource();
-		File swaggerFile = new File(filePath);
-		assertTrue(swaggerFile.exists());
+		final Swagger swagger = getSwaggerFile(filePath);
+		assertNotNull(swagger);
 
-		try (InputStream openapiDoc = new FileInputStream(swaggerFile)) {
-			String rawText = readUTF8(openapiDoc);
+		String title = "Example";
+		if (swagger.getInfo() != null && swagger.getInfo().getTitle() != null) {
+			title = swagger.getInfo().getTitle();
+		}
+		String artifactId = title.toLowerCase().replaceAll("[^a-zA-Z0-9-_]+", "-").replaceAll("-+", "-");
 
-			final Swagger swagger = new SwaggerParser().read(Json.mapper().readTree(rawText));
+		HashMap<String, Object> variables = new HashMap<>();
+		variables.put("swagger", swagger);
+		variables.put("title", title);
+		variables.put("artifactId", artifactId);
+		variables.put("fuseVersion", GenerateFuseProjectResource.FUSE_VERSION);
 
-			String title = "Example";
-			if (swagger.getInfo() != null && swagger.getInfo().getTitle() != null) {
-				title = swagger.getInfo().getTitle();
-			}
-			String artifactId = title.toLowerCase().replaceAll("[^a-zA-Z0-9-_]+", "-").replaceAll("-+", "-");
+		return generator.generateCamelContextXML(variables);
+	}
 
-			HashMap<String, Object> variables = new HashMap<>();
-			variables.put("swagger", swagger);
-			variables.put("title", title);
-			variables.put("artifactId", artifactId);
-			variables.put("fuseVersion", GenerateFuseProjectResource.FUSE_VERSION);
+	private Swagger getSwaggerFile(String filePath) throws Exception {
+		File swaggerFile = new File(filePath);
+		assertTrue(swaggerFile.exists());
 
-			return generator.generateCamelContextXML(variables);
+		try (InputStream openapiDoc = new FileInputStream(swaggerFile)) {
+			String rawText = readUTF8(openapiDoc);
+	        Swagger swagger = 
+	        		new SwaggerParser().read(Json.mapper().readTree(rawText));
+			return swagger;
 		}
 	}
 
+	private Swagger getCleanedUpSwagger(String filePath) throws Exception {
+		GenerateFuseProjectResource generator = new GenerateFuseProjectResource();
+		final Swagger initialSwagger = getSwaggerFile(filePath);
+	    final Swagger fixedSwagger = generator.cleanupSwaggerFile(initialSwagger);
+		return fixedSwagger;
```

Comment:
I think we can return directly the result instead of using the local variable fixedSwagger. What do you think?